playbooks/common/openshift-cluster/openshift_metrics.yml

```diff
@@ -1,4 +1,11 @@
 ---
+# I believe this violates our rule of either includes or plays.
+# Unfortunately we've documented calling this playbook directly so
+# I'd really like to apply the group mappings here rather than up in byo
+- include: ../../common/openshift-cluster/std_include.yml
```

Comment:
We're doing it in `playbooks/byo/openshift-cluster/openshift_logging.yml`.

Unfortunately docs.openshift.com/org instructs metrics users to call this playbook rather than `playbooks/byo/openshift-cluster/openshift_metrics.yml` I guess I could just make them the same and we can deal with folks having to define oo_first_master for metrics until we release an errata and also update the docs to tell people to use `byo/....`